src/promptflow/promptflow/_sdk/operations/_local_azure_connection_operations.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, connection_provider, **kwargs):
         # Lazy init client as ml_client initialization require workspace read permission
         self._pfazure_client = None
         self._credential = self._get_credential()
+        self._user_agent = kwargs.pop("user_agent", None)
```

Comment:
Not sure is local PFS should add it's own UA like "local-pfs/xxx"

Reply:
added actually